interface-definitions/nat.xml.in

```diff
@@ -18,12 +18,13 @@
                 <properties>
                   <help>Inbound interface of NAT traffic</help>
                   <completionHelp>
+                    <list>any</list>
                     <script>${vyos_completion_dir}/list_interfaces.py</script>
                   </completionHelp>
                 </properties>
               </leafNode>
               <node name="translation">
-                <properties>
+                <properties>     
```

Comment:
Please remove the addditional whitespaces.

Reply:
Removed it, thank you